packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -2,16 +2,13 @@
 
 :local {
   .profile {
-    background: var(--terra-consumer-profile-link-border-background, url('http://d320j1w59ws5w0.cloudfront.net/terra-consumer-background-images.jpg'));
+    background: var(--terra-consumer-profile-link-border-background, #c7d4ea url('http://d320j1w59ws5w0.cloudfront.net/terra-consumer-background-images.jpg'));
     background-attachment: fixed;
     background-size: cover;
-    border-top: var(--terra-consumer-profile-link-border-top, 3px solid #c7d3ea);
     margin-top: 5px;
 
-    @supports (-webkit-overflow-scrolling: touch) {
-      background: none;
-      background-attachment: none;
-      background-size: none;
+    @media screen and (max-width: $terra-medium-breakpoint) {
```

Comment:
small screen sizes never has a fixed background behind the menu, so the profile menu never should on small screens either

Reply:
@amf9t2 ^^

Reply:
couple points I see after this change
1. Ipad landscape view is same as desktop. 
2. Mobile view has different background in content area and navigation except ipad landscape.

Reply:
discussed with @mhemesath due to issue in links because of background image we will continue with this change.